providers/nodeReleasesProvider.tsx

```diff
@@ -0,0 +1,28 @@
+import { createContext, useCallback } from 'react';
+import { useFetchNodeReleases } from '../hooks/useFetchNodeReleases';
+import type { FC, PropsWithChildren } from 'react';
+import type { NodeRelease, NodeReleaseStatus } from '../types';
+
+export const NodeReleasesContext = createContext<{
+  releases: NodeRelease[];
+  getReleaseByStatus: (status: NodeReleaseStatus) => NodeRelease | undefined;
+}>({
+  releases: [],
+  getReleaseByStatus: (_: NodeReleaseStatus) => undefined,
```

Comment:
I wasn't thinking that this should be part of the context 😅 

Usually it's an anti-pattern to add a function to a context, the idea was that this function could be invoked from the hook itself, not the provider... 👀

Reply:
Thanks!

I didn't think about it when I was implementing. Let me move it to the Hook (just checked, `useLocale` also implements the function in Hook rather than Provider).